core/control-plane/control-plane-api/src/main/java/org/eclipse/dataspaceconnector/controlplane/api/transferprocess/model/TransferProcessFailStateDto.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (c) 2020, 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.controlplane.api.transferprocess.model;
+
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+@JsonDeserialize(builder = TransferProcessFailStateDto.Builder.class)
+public class TransferProcessFailStateDto {
```

Comment:
Seems like this class and `TransferProcessFailRequest` are redundant. could we delete one?

Reply:
We could i only didn't know where to put this common payload class. 
The two modules `control-plane-api` and `control-plane-api-client` don't share a common module so i was not sure

Reply:
Ah, that's right. 
Technically there should be a shared module, something like a `control-plane-api-model`, but for now that would only contain the `TransferProcessFailStateDto`. Not sure that is worth it. That could be revisited, should the control plane's internal API ever become "wider".
So I guess we could leave it for now.